src/pages/Home.module.css

```diff
@@ -1,14 +1,58 @@
-.hero{
-    height:17rem;
+.hero {
+  height: 17rem;
 }
+
 .btn {
-    border-radius:var(--border-radius1);
-    color: var(--secondry-color);
-    font-size: var(--size-1);
-    background: var(--red-colorl1);
-    padding: 10px 20px 10px 20px;
-    text-decoration: none;
-  }
-  .btn:hover {
-    background: var(--red-colord2);
-  }
\ No newline at end of file
+  border-radius: var(--border-radius1);
+  color: var(--secondry-color);
+  font-size: var(--size-1);
+  background: var(--red-colorl1);
+  padding: 10px 20px 10px 20px;
+  text-decoration: none;
+}
+.btn:hover {
+  background: var(--red-colord2);
+}
+.searchbar input {
+  min-width: 30rem;
+  height: 3rem;
+  border: 2px solid transparent;
+  border-radius: var(--border-radius);
+  background: var(--input-box-color);
+  display: flex;
+}
+/* aside */
+.routelink {
+  text-decoration: none;
+  color: var(--black-color);
+}
+.aside {
+  width: 18rem;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  min-height: auto;
+
+  letter-spacing: 0.01785714em;
+  font-family: "Google Sans", Roboto, Arial, sans-serif;
```

Comment:
If this font family is global, then you can apply it to either root or reset CSS instead